### ISSUE TITLE
Windows installer

### ISSUE THREAD

**Issue by moigagoo** [minbrowser/min#359]
First off, I just want to say a huge thank you for your work on Min. I use it as my full-time browser on macOS and I find the experience magnificent. When I first tried it, I didn't really think it could replace Chrome or Safari, but it proved me wrong.

I would love to use it on Windows as well. The Windows build seems to work all right, but because the browser is not installed to the system, I can't set it as the default one.

**Comment by PalmerAL** [comment]
Glad to hear you like it!

There was some work on a windows installer in the <a href="https://github.com/minbrowser/min/tree/windows">windows</a> branch, I'm not sure if it still works. I'll look into this soon and see what would be required to finish that.

**Comment by SweRavn** [comment]
@PalmerAL: What is the Windows branch? I managed to download the Windows binary, but after forking and checking out the code, I did not manage to run Min from the code. I get the following error:
```
npm install
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code EBADPLATFORM

npm ERR! notsup Unsupported platform for grunt-electron-installer-debian@0.4.0: wanted {"os":"darwin,linux","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm ERR! notsup Valid OS:    darwin,linux
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: x64

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\ravn\Project\webbrowser\minbrowser\min\npm-debug.log
```

I would very much like to contribut to this projects, but this is probably the first thing I need to sort out.

**Comment by PalmerAL** [comment]
@SweRavn Sorry about that! Fixed in 77170e38c603bda7ec583c029017b587552c7754.

There are a couple different places with windows-specific code:

 - The current master branch works on Windows, but there isn't a way to build an installer (so when you download Min, you get an exe and a bunch of other files in a folder). Also, the window frame doesn't look very "native".

 - The ```windows``` branch (https://github.com/minbrowser/min/tree/windows) has mostly-working code in ```gruntfile.js``` and ```main/main.js``` to build an installer and add a Min shortcut to the desktop when the app is first launched. I started working on this a long time ago, and then got sidetracked by other things.

 - PR #369 has code to make the titlebar look more like a native app on Windows, and also to make keyboard shortcuts better match native Windows applications.

Eventually, the plan is to merge both of these into the master branch, and when that is done Min should work mostly like a native Windows app.

I realize this is pretty confusing, so don't hesitate to ask if you have questions!
